### PR TITLE
Updating the prediction menu

### PR DESCRIPTION
**Pull request by jmaslek**
I am cleaning up the prediction menu.

Will update this if I add new features.

### CODE REVIEW

File: gamestonk_terminal/config_neural_network_models.py [hunk 1]
```diff
@@ -23,19 +23,36 @@
...
     {"SimpleRNN": {"units": 21, "activation": "linear", "return_sequences": False}},
-    {"Dense": {"activation": "linear"}},
+    {"Dense": {"activation": "relu"}},
 ]
 
 Long_Short_Term_Memory = [
     {"LSTM": {"units": 25, "activation": "tanh", "return_sequences": True}},
     {"LSTM": {"units": 15, "activation": "tanh", "return_sequences": False}},
-    {"Dense": {"activation": "linear"}},
+    {"Dense": {"activation": "relu"}},
```
Comment (DidierRLopes) [inline gamestonk_terminal/config_neural_network_models.py:40]: This should close #45 right?

File: gamestonk_terminal/main_helper.py [hunk 2]
```diff
@@ -147,7 +147,7 @@ def load(l_args, s_ticker, s_start, s_interval, df_stock):
         "-s",
         "--start",
         type=valid_date,
-        default="2015-01-01",
+        default="2019-01-01",
```
Comment (DidierRLopes) [inline gamestonk_terminal/main_helper.py:150]: I don't think this was supposed, right?

File: gamestonk_terminal/config_neural_network_models.py [hunk 3]
```diff
@@ -23,19 +23,36 @@
...
+    {"MaxPool1D": {"pool_size": 2}},
+    {"Conv1D": {"filters": 10, "kernel_size": 5, "activation": "relu"}},
+    {"MaxPool1D": {"pool_size": 2}},
+    {"Flatten": {}},
+    {"Dense": {"activation": "relu"}},
+]
+
+# Optimizers :  https://www.tensorflow.org/api_docs/python/tf/keras/optimizers
+# Adam, Adagrad, Adadelta, Adamax,Ftrl, Nadam, Rmsprop, Sgd
+Optimizer = "Adam"
```
Comment (DidierRLopes) [inline gamestonk_terminal/config_neural_network_models.py:54]: What do you think about also having:
* Learning Rate
* Loss Function
* Pre-processing technique
* Training Epochs

And we could leave for command line just: Input days, Prediction days

File: gamestonk_terminal/prediction_techniques/README.md [hunk 4]
```diff
@@ -153,23 +142,25 @@ Facebook's Prophet:
 
 ## mlp <a name="mlp"></a>
 ```
-usage: mlp [-d N_DAYS] [-i N_INPUTS] [-j N_JUMPS] [--epochs N_EPOCHS] [-p {normalization,standardization,none}]
-[-o {adam,adagrad,adadelta,adamax,ftrl,nadam,optimizer,rmsprop,sgd}] [-l {mae,mape,mse,msle}] [-e S_END_DATE] [--loops N_LOOPS]
+usage: mlp [-d N_DAYS] [-i N_INPUTS] [-j N_JUMPS] [--epochs N_EPOCHS] [-p {minmax, normalization,standardization,none}]
```
Comment (DidierRLopes) [inline gamestonk_terminal/prediction_techniques/README.md:145]: jump can be removed

File: gamestonk_terminal/prediction_techniques/README.md [hunk 5]
```diff
@@ -153,23 +142,25 @@ Facebook's Prophet:
...
+ [-l {mae,mape,mse,msle}] [-e S_END_DATE] [--loops N_LOOPS] [-v VALID] [--lr LEARNING_RATE] [--no_shuffle]
 ```
 MulitLayer Perceptron:
   * -d : prediciton days. Default 5.
   * -i : number of days to use for prediction. Default 40.
   * -j : number of jumps in training data. Default 1.
```
Comment (DidierRLopes) [inline gamestonk_terminal/prediction_techniques/README.md:151]: jump can be removed

File: gamestonk_terminal/prediction_techniques/README.md [hunk 6]
```diff
@@ -183,30 +174,32 @@ MultiLayer_Perceptron \
...
 ## rnn <a name="rnn"></a>
 ```
 usage: rnn [-d N_DAYS] [-i N_INPUTS] [-j N_JUMPS] [--epochs N_EPOCHS] [-p {normalization,standardization,none}]
-[-o {adam,adagrad,adadelta,adamax,ftrl,nadam,optimizer,rmsprop,sgd}] [-l {mae,mape,mse,msle}] [-e S_END_DATE] [--loops N_LOOPS]
+ [-l {mae,mape,mse,msle}] [-e S_END_DATE] [--loops N_LOOPS] [-v VALID] [--lr LEARNING_RATE] [--no_shuffle]
 ```
 Recurrent Neural Network:
   * -d : prediciton days. Default 5.
   * -i : number of days to use for prediction. Default 40.
   * -j : number of jumps in training data. Default 1.
```
Comment (DidierRLopes) [inline gamestonk_terminal/prediction_techniques/README.md:190]: jump can be removed

File: gamestonk_terminal/prediction_techniques/README.md [hunk 7]
```diff
@@ -183,30 +174,32 @@ MultiLayer_Perceptron \
...
         {'Dense':
-                {'activation':'linear'} }]
+                {'activation':'relu'} }]
 ```
 
 ![mlp](https://user-images.githubusercontent.com/25267873/108604944-d2025400-73a8-11eb-9ab6-52972160cd2a.png)
 
 ## rnn <a name="rnn"></a>
 ```
 usage: rnn [-d N_DAYS] [-i N_INPUTS] [-j N_JUMPS] [--epochs N_EPOCHS] [-p {normalization,standardization,none}]
```
Comment (DidierRLopes) [inline gamestonk_terminal/prediction_techniques/README.md:184]: jump can be removed

File: gamestonk_terminal/prediction_techniques/README.md [hunk 8]
```diff
@@ -235,22 +228,24 @@ Recurrent_Neural_Network \
 ## lstm <a name="lstm"></a>
 ```
 usage: lstm [-d N_DAYS] [-i N_INPUTS] [-j N_JUMPS] [--epochs N_EPOCHS] [-p {normalization,standardization,none}]
```
Comment (DidierRLopes) [inline gamestonk_terminal/prediction_techniques/README.md:230]: jump can be removed

File: gamestonk_terminal/prediction_techniques/README.md [hunk 9]
```diff
@@ -235,22 +228,24 @@ Recurrent_Neural_Network \
...
-[-o {adam,adagrad,adadelta,adamax,ftrl,nadam,optimizer,rmsprop,sgd}] [-l {mae,mape,mse,msle}] [-e S_END_DATE] [--loops N_LOOPS]
+ [-l {mae,mape,mse,msle}] [-e S_END_DATE] [--loops N_LOOPS] [-v VALID] [--lr LEARNING_RATE] [--no_shuffle]
 ```
 Long-Short Term Memory:
   * -d : prediciton days. Default 5.
   * -i : number of days to use for prediction. Default 40.
   * -j : number of jumps in training data. Default 1.
```
Comment (DidierRLopes) [inline gamestonk_terminal/prediction_techniques/README.md:236]: jump can be removed

File: gamestonk_terminal/prediction_techniques/README.md [hunk 10]
```diff
@@ -273,3 +268,24 @@ Long_Short_Term_Memory \
...
 
+## conv1d <a name="conv1d"></a>
+```
+usage: lstm [-d N_DAYS] [-i N_INPUTS] [-j N_JUMPS] [--epochs N_EPOCHS] [-p {normalization,standardization,minmax,none}]
+ [-l {mae,mape,mse,msle}] [-e S_END_DATE] [--loops N_LOOPS] [-v VALID] [--lr LEARNING_RATE] [--no_shuffle]
+```
+1D Convolutional Neural Net:
+  * -d : prediction days. Default 5.
+  * -i : number of days to use for prediction. Default 40.
+  * -j : number of jumps in training data. Default 1.
```
Comment (DidierRLopes) [inline gamestonk_terminal/prediction_techniques/README.md:279]: jump can be removed

File: gamestonk_terminal/prediction_techniques/README.md [hunk 11]
```diff
@@ -273,3 +268,24 @@ Long_Short_Term_Memory \
 
 <img width="992" alt="Captura de ecrã 2021-03-14, às 00 06 51" src="https://user-images.githubusercontent.com/25267873/111053158-420c7300-8459-11eb-993b-6d9c26f98af9.png">
 
+## conv1d <a name="conv1d"></a>
+```
+usage: lstm [-d N_DAYS] [-i N_INPUTS] [-j N_JUMPS] [--epochs N_EPOCHS] [-p {normalization,standardization,minmax,none}]
```
Comment (DidierRLopes) [inline gamestonk_terminal/prediction_techniques/README.md:273]: usage: conv1d AND jump can be removed

Comment (DidierRLopes) [reply gamestonk_terminal/prediction_techniques/README.md:273]: If epochs, preprocessing, lossfunction and learning rate end up being set on config_neural_network. Then we can remove them from all the functions here.

**Review by DidierRLopes — COMMENTED** [review]
Niceeeee!!! I added some few suggestions, but great work!

File: gamestonk_terminal/config_neural_network_models.py (hunk 1 above)
Comment (jmaslek) [reply gamestonk_terminal/config_neural_network_models.py:40]: In theory the relu should suppress anything < 0.

File: gamestonk_terminal/main_helper.py (hunk 2 above)
Comment (jmaslek) [reply gamestonk_terminal/main_helper.py:150]: As we chatted about, I think 2 years of data by default is fine.

File: gamestonk_terminal/config_neural_network_models.py (hunk 3 above)
Comment (jmaslek) [reply gamestonk_terminal/config_neural_network_models.py:54]: So I think LR and Epochs should be command line (so you don't have to restart every time if you wanna change one of them).  Loss and preprocess is probably good!

**Review by DidierRLopes — APPROVED** [review]
